tests/test_unit_geometry.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+import numpy as np
+from manim import Sector, ORIGIN
+
+
+def test_get_arc_center():
+    assert np.all(Sector().get_arc_center() == ORIGIN)
```

Comment:
This test also passes without your patch to the function: if the intersection in `get_arc_center` fails, then it always returns `ORIGIN`. Please add a test for a sector/arc whose center is not `ORIGIN`.